pkg/tbtc/chain.go

```diff
@@ -120,6 +121,58 @@ type DistributedKeyGenerationChain interface {
 	DKGParameters() (*DKGParameters, error)
 }
 
+// InactivityClaimedEvent represents an inactivity claimed event. It is emitted
+// after a submitted inactivity claim lands on the chain.
+type InactivityClaimedEvent struct {
+	WalletID    [32]byte
+	Nonce       *big.Int
+	Notifier    chain.Address
+	BlockNumber uint64
+}
+
+// InactivityChainClaim represents an inactivity claim submitted to the chain.
+type InactivityChainClaim struct {
```

Comment:
I think we are good to name it just:
```suggestion
type InactivityClaim struct {
```

Reply:
Done in 0e3b9e5a5e7bfe2b010ecf77509bdfe5536713a2.